### PR TITLE
Update --client-ca-file in KubeletClientCa check (CKV_K8S_140)

### PR DESCRIPTION
**Pull request by nijdarshan**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description

- The parameter `--client-ca-file` in the KubeletClientCa.py is not checked, instead `--root-ca-file` is checked. So raising a PR with the correct parameter.

- Kubernetes Check # `CKV_K8S_102`

Fixes # (issue)
- `--root-ca-file` to `--client-ca-file`
- Also, updated the examples.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/kubernetes/checks/resource/k8s/ApiServerEtcdCaFile.py [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
+        name = "Ensure that the --etcd-cafile argument is set as appropriate"
         super().__init__(name=name, id=id)
 
     def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) -> CheckResult:
         self.evaluated_container_keys = ["command"]
         keys, values = extract_commands(conf)
 
         if "kube-apiserver" in keys:
-            if "--etcd-ca-file" not in keys:
+            if "--etcd-cafile" not in keys:
```
Comment (gruebel) [inline checkov/kubernetes/checks/resource/k8s/ApiServerEtcdCaFile.py:19]: for some reason your old changes are still in this PR, can you please remove them?

File: tests/kubernetes/checks/example_ApiServerEtcdCaFile/example_ApiServerEtcdCaFile-PASSED.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ spec:
   containers:
   - command:
     - kube-apiserver
-    - --etcd-ca-file=ca.file
+    - --etcd-cafile=ca.file
```
Comment (gruebel) [inline tests/kubernetes/checks/example_ApiServerEtcdCaFile/example_ApiServerEtcdCaFile-PASSED.yaml:14]: same here

File: checkov/kubernetes/checks/resource/k8s/KubeletClientCa.py [hunk 3]
```diff
@@ -13,16 +13,14 @@ def __init__(self) -> None:
 
     def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) -> CheckResult:
         self.evaluated_container_keys = ["command"]
-        if conf.get("command"):
+        if conf.get("command") is not None:
```
Comment (gruebel) [inline checkov/kubernetes/checks/resource/k8s/KubeletClientCa.py:16]:
```suggestion
        if conf.get("command"):
```
no need to especially check for `None` the previous condition checks for `is not None`  and also that it is not empty

**Review by gruebel — COMMENTED** [review]
nice, looks good so far 💪 please remove the changes to the other check, which are already merged anyway 🙂

File: checkov/kubernetes/checks/resource/k8s/ApiServerEtcdCaFile.py (hunk 1 above)
Comment (nijdarshan) [reply checkov/kubernetes/checks/resource/k8s/ApiServerEtcdCaFile.py:19]: I'm not sure if I can remove them I will have to raise the PR again. I shall do that!

**Pull request closed without merging**